cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,79 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
+  
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables')
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+
+  it('should have the ability pagination', () => {
+    cy.get('.-pagination').should('contain', 'Previous')
+      .and('contain', 'Next');
+    cy.contains('.-pageInfo', 'Page').should('exist');
```

Comment:
you do not check the pagination here. You need to add workers to make workers amount equal to 6. Than Change rows count to 5. Click the button next -> assert that page 2 is displayed. Click button previous -> assert that page 1 is displayed.

Reply:
I fixed all